src/main/java/edu/ucar/unidata/rosetta/converters/TagUniversalFileFormat.java

```diff
@@ -399,9 +399,9 @@ private String convertOneLocOneOb(String ncfile) throws InvalidRangeException {
             theNewVar.addAttribute(new Attribute("units", timeUnit));
 
             // create variables:
-            List sortedKeys = new ArrayList(data.keySet());
+            List<String> sortedKeys = new ArrayList<String>(data.keySet());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)